Source/PlasticSourceControl/Private/PlasticSourceControlMenu.cpp

```diff
@@ -474,10 +488,49 @@ void FPlasticSourceControlMenu::OnSourceControlOperationComplete(const FSourceCo
 					UE_LOG(LogSourceControl, Log, TEXT("Reload: %s"), *PackageName);
 				}
 			}
-			// else, it means the file is not an asset from the Content/ folder (eg config, source code, anything else)
+			// else, it means the file is not an asset from the Content/ folGet the World currently loaded by the Editorder (eg config, source code, anything else)
+		}
+
+		// Detects if some packages to reload are par of the current map
+		// (ie assets within __ExternalActors__ or __ExternalObjects__ from the new One File Per Actor (OFPA) in UE5)
+		// in which case the current map need to be reloaded, so it needs to be added to the list of packages if not already there
+		// (then UPackageTools::ReloadPackages() will handle unloading the map at the start of the reload, avoiding some crash, and reloading it at the end)
+		if (UWorld* CurrentWorld = GetCurrentWorld())
+		{
+			bool bNeedReloadCurrentMap = false;
+			UPackage* CurrentMapPackage = CurrentWorld->GetOutermost();
+			const FString CurrentMapPath = *CurrentMapPackage->GetName();					// eg "/Game/Maps/OpenWorld"
+			const FString CurrentMapPathWithoutGamePrefix = CurrentMapPath.RightChop(5);	// eg "/Maps/OpenWorld"
```

Comment:
Could this magic number `5` be extracted to a constant or something?
Something like `FString("/Game").length()` would be clearer.

In fact, as the string "/Game/" is already declared in line 504, we could reuse it.

```
const FString GamePath = FString("/Game");

const FString CurrentMapPathWithoutGamePrefix = CurrentMapPath.RightChop(GamePath.length());

const FString CurrentMapExternalActorPath = GamePath + "/" + FPackagePath::GetExternalAct...
```